scvi/models/log_likelihood.py

```diff
@@ -47,29 +47,33 @@ def compute_reconstruction_error(vae, posterior, **kwargs):
 
 def compute_marginal_log_likelihood(vae, posterior, n_samples_mc=100):
     """ Computes a biased estimator for log p(x), which is the marginal log likelihood.
-
-    Despite its bias, the estimator still converges to the real value
-    of log p(x) when n_samples_mc (for Monte Carlo) goes to infinity
-    (a fairly high value like 100 should be enough)
-    Due to the Monte Carlo sampling, this method is not as computationally efficient
-    as computing only the reconstruction loss
+        Despite its bias, the estimator still converges to the real value
+        of log p(x) when n_samples_mc (for Monte Carlo) goes to infinity
+        (a fairly high value like 100 should be enough)
+        Due to the Monte Carlo sampling, this method is not as computationally efficient
+        as computing only the reconstruction loss
     """
-    # Uses MC sampling to compute a tighter lower bound on log p(x)
+    # Uses MC sampling to compute a tighter lower bound
     log_lkl = 0
     for i_batch, tensors in enumerate(posterior):
         sample_batch, local_l_mean, local_l_var, batch_index, labels = tensors
         x = torch.log(1 + sample_batch)
```

Comment:
is this not useful anymore ?

Reply:
Indeed, the "x = log(1 + sample_batch)" is performed and passed to z_encoder in vae.inference, like done manually in the previous code. I think this is another reason to use vae.inference instead of vae.forward